pkg/expr/sql/parser.go

```diff
@@ -0,0 +1,28 @@
+package sql
+
+import (
+	"errors"
+
+	"github.com/xwb1989/sqlparser"
```

Comment:
Seems this is for MySQL, but this is the DuckDB dialect?

Reply:
My understanding is that they [try to keep it compatible](https://news.ycombinator.com/item?id=24531894).

Reply:
We are only capturing table names, so we know what the Refs are.  If this parser becomes a problem I think we can write a simple tokenizer to capture the table names.

Reply:
I think this is maybe why `SELECT [1, 2, 3];` returns invalid SQL, was trying to create some structures.

Reply:
maybe `SELECT array_value(1, 2, 3)`

Reply:
@kylebrandt that should work now I made the parser a bit more lenient.